zebra-consensus/src/chain.rs

```diff
@@ -148,7 +149,11 @@ where
     }
 }
 
-/// Initialize block and transaction verification services.
+/// Initialize block and transaction verification services,
+/// and pre-download Groth16 parameters if needed.
```

Comment:
Nit: should it just be `download`?

```suggestion
/// and download Groth16 parameters if needed.
```

Reply:
We pre-download the parameters as soon as Zebra starts up, to avoid timeouts when full verification starts.

Explained in commit e74eea922.